README.md

```diff
@@ -1,36 +1,53 @@
 # Documentation details
 
-Contracting processes involve many different documents. There should be a presumption of 'open by default' when it comes to publishing these documents as part of an Open Contracting process.
-
 The OCDS documentation block, which is used widely through the different sections of the standard, provides a means to link out to externally hosted documents.
 
-These should generally be provided at their own stable URL, without requirements to log-in or authenticate. Redactions should be kept to a minimum.
-
 ## Providing enhanced documentation and document links
 
 A number of implementations of OCDS, notable OCDS for PPPs, explicitly require a number of key items of information to be disclosed. These items are articulated as an extended list of entries for the `documentType` codelist.
 
 However, rather than just linking out to the entire document where the information is found, this extension describes a way to:
```

Comment:
This is the key content, and should be re-written as the opening paragraph describing the extension.